### DISCUSSION TITLE
Seeing warn log in window_op.go after upgrading to v2.0.4

### DISCUSSION THREAD

**Discussion by ankit-4129** [lf-edge/ekuiper#3442] [Q&A] [answered]
I upgraded kuiper from v1.13.3 to v2.0.4 everything is working as expected but I am seeing this warn logs.
```
time="2024-12-12T11:51:24Z" level=warning msg="triggerTime: 1734004284000, length: 60000000000, delta: 0, leftmost: 1734004224000" file="node/window_op.go:572" rule=rulename***
```
I believe that this log was set to warn by mistake, it should probably be at level debug/trace.
https://github.com/lf-edge/ekuiper/blob/85af151beee5228481db66ec3f1b7201eba73cad/internal/topo/node/window_op.go#L572

Can someone please confirm this?

**Comment by ngjaying** [comment, marked as answer]
@ankit-4129 Yeah, this is accidentally added for debug. We will revert it in the next fix pack. Thanks for reporting
